rclpy/rclpy/__init__.py

```diff
@@ -32,6 +32,8 @@ def init(*, args=None):
 def create_node(node_name, *, namespace=None):
     namespace = namespace or ''
     failed = False
+    if not _rclpy.rclpy_is_initialized():
+        raise NotInitializedException('rclpy has not been initialized')
```

Comment:
Since the exception type `rclpy.NotInitializedException` already encapsulates the semantic I would not pass an additional string message.